### PR TITLE
Prepare for packaging

### PR DESCRIPTION
**Pull request by JonathanDCohen**
After merge, this PR allows netanalysis to be installed with 

`pip install git+git://github.com/Jigsaw-Code/net-analysis.git@master`

It may be necessary to first install `setuptools` and `wheel` first

### CODE REVIEW

File: python_env.md [hunk 1]
```diff
@@ -4,33 +4,28 @@
...
-Tools for development should be listed in [py_dev_requirements.txt](py_dev_requirements.txt).
+Libraries used in code should be listed in [setup.py](setup.py).
 
-To install the new dependencies without re-creating the Python virtual environment, run
-
-```
-./pipenv.sh install -r py_dev_requirements.txt
-```
\ No newline at end of file
+Tools for development should be listed in the [Pipfile](Pipfile).
```
Comment (fortuna) [inline python_env.md]: Maybe mention that they should run the installation again after changing the files.

File: MANIFEST.in [hunk 2]
```diff
@@ -14,7 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Sets up the Python environment for development under .venv
-
-rm -f Pipfile
-PIPENV_VENV_IN_PROJECT=1 pipenv --python="3.7" install -r py_dev_requirements.txt
+graft netanalysis/third_party
```
Comment (fortuna) [inline MANIFEST.in]: Maybe add a comment on what this does. I think people are not familiar with `MANIFEST.in`

File: netanalysis/infrastructure/resources.py [hunk 3]
```diff
@@ -21,7 +21,7 @@
 assert (_PACKAGE_DIR.split(os.path.sep)[-2:] == ["netanalysis", "infrastructure"]), \
     "resources.py in invalid directory %s" % _PACKAGE_DIR
 
-_RESOURCES_ROOT = os.path.normpath(os.path.join(_PACKAGE_DIR, "..", ".."))
+_RESOURCES_ROOT = os.path.normpath(os.path.join(_PACKAGE_DIR, ".."))
```
[the remark this answers is not in the document]
Comment (JonathanDCohen) [reply netanalysis/infrastructure/resources.py]: third_party was moved to inside netanalysis/ so it's within the package structure when installed

File: MANIFEST.in (hunk 2 above)
Comment (JonathanDCohen) [reply MANIFEST.in]: Done in 9d6eac5

File: python_env.md (hunk 1 above)
Comment (JonathanDCohen) [reply python_env.md]: Done in 9d6eac5

File: netanalysis/infrastructure/resources.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (JonathanDCohen) [reply netanalysis/infrastructure/resources.py]: After speaking offline, done using a symlink in 67ec435

File: MANIFEST.in (hunk 2 above)
Comment (JonathanDCohen) [reply MANIFEST.in]: Also, this is a very weird diff

File: third_party/third_party [hunk 4]
```diff
@@ -0,0 +1 @@
+third_party
```
Comment (fortuna) [inline third_party/third_party]: Do we need this? What is it doing?

Comment (JonathanDCohen) [reply third_party/third_party]: Not sure how that got there.  I tested from scratch with my local clone removed and everything was good to go.

**Pull request merged**